week4_approx/dqn_atari.ipynb

```diff
@@ -47,7 +47,7 @@
    "metadata": {},
    "source": [
     "### Let's play some old videogames\n",
-    "![img](https://s17.postimg.cc/y9xcab74f/nerd.png)\n",
+    "![img](https://github.com/yandexdataschool/Practical_RL/raw/master/yet_another_week/_resource/nerd.png)\n",
```

Comment:
Don't we need to host images somewhere else?

Reply:
We do, but that's out of scope for this PR.